elt-pipelines/facility_ops/ingest/accelerator/statusdisplay/statusdisplay.py

```
@@ -0,0 +1,62 @@
+
+
+def extract_cycles(_):
+    data = clean(fetch())
+    newline_delimited = "\n".join(json.dumps(row) for row in data)
+
+    with io.BytesIO(newline_delimited.encode()) as f:
+        yield pyarrow.json.read_json(f)
+
+
+def fetch():
+    response = requests.get(CYCLES_URL)
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)